### PR TITLE
Add service registry

### PR DESCRIPTION
**Pull request by yk-sgr**

### CODE REVIEW

File: src/main/kotlin/wtf/votebot/bot/ApplicationInfo.kt [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see https://www.gnu.org/licenses/.
+ */
+
+package wtf.votebot.bot
+
+class ApplicationInfo {
+    companion object {
+        // TODO const val RELEASE = "@RELEASE@"
```
Comment (DRSchlaubi) [inline src/main/kotlin/wtf/votebot/bot/ApplicationInfo.kt:24]: Done in #7 :D

File: src/main/kotlin/wtf/votebot/bot/config/Config.kt [hunk 2]
```diff
@@ -37,6 +39,6 @@ interface Config {
     }
 
     fun environmentType(): Environment {
-        return Environment.valueOf(environment.toLowerCase())
```
Comment (DRSchlaubi) [inline src/main/kotlin/wtf/votebot/bot/config/Config.kt:40]: Why not using single line methods here (in the whole file): https://subscription.packtpub.com/book/application_development/9781787126367/4/ch04lvl1sec49/single-expression-functions

**Review by DRSchlaubi — CHANGES_REQUESTED** [review]
Why not using single line methods in

File: src/main/kotlin/wtf/votebot/bot/config/Config.kt (hunk 2 above)
Comment (yk-sgr) [reply src/main/kotlin/wtf/votebot/bot/config/Config.kt:40]: Done

**Review by DRSchlaubi — APPROVED** [review]

**Pull request merged**